### ISSUE TITLE
cmd/cue: docs for "cue help fix" incorrectly say its default operation is module-wide

### ISSUE THREAD

**Issue by jpluscplusm** [cue-lang/cue#3438]
### What version of CUE are you using (`cue version`)?

<pre>
$ cue version
cue version v0.10.0

go version go1.23.0
      -buildmode exe
       -compiler gc
       -trimpath true
     CGO_ENABLED 0
          GOARCH amd64
            GOOS linux
         GOAMD64 v1
cue.lang.version v0.10.0

</pre>

### Does this issue reproduce with the latest stable release?

v0.10.0 is latest.

### What did you do?

I read https://cuelang.org/docs/reference/command/cue-help-fix/ and found:

> Without any packages, fix applies to all files within a module.

This does not appear to be the case, as this test shows:

```
exec cue mod init
exec cue fix
cmp file.cue file.gold
cmp dir/file.cue file.gold
-- file.cue --
x: 5 mod 3
-- dir/file.cue --
x: 5 mod 3
-- file.gold --
x: __mod(5, 3)
```

### What did you expect to see?

A passing test.

### What did you see instead?

```
> exec cue mod init
> exec cue fix
> cmp file.cue file.gold
diff file.cue file.gold
--- file.cue
+++ file.gold
@@ -1,1 +1,1 @@
-x: 5 mod 3
+x: __mod(5, 3)

FAIL: /tmp/testscript914189743/cue.fix.defaultIsNotModuleWide.txtar/script.txtar:3: file.cue and file.gold differ
> cmp dir/file.cue file.gold
diff dir/file.cue file.gold
--- dir/file.cue
+++ file.gold
@@ -1,1 +1,1 @@
-x: 5 mod 3
+x: __mod(5, 3)

FAIL: /tmp/testscript914189743/cue.fix.defaultIsNotModuleWide.txtar/script.txtar:4: dir/file.cue and file.gold differ
```

**Comment by jpluscplusm** [comment]
I opened this prematurely: the command's operation **is** module-wide, but this necessarily excludes CUE files without a package statement. I'll follow up by updating or closing this issue soon.

**Comment by mvdan** [comment]
Sorry for the confusion here. You're right that the docs are correct - `cue fix` works on the entire module with no args. The fact that you got thrown off by package-less files is a different cue fix bug - which we fixed in https://github.com/cue-lang/cue/issues/3405 for the upcoming v0.11 alpha :)

**Comment by mvdan** [comment]
Yup, confirmed that your original testscript passes as of bb24c7ce1f048a01cba5ae7dcfe0813e127cfd42. Closing as a duplicate of https://github.com/cue-lang/cue/issues/3405.

Do you think you need the fix from 3405 backported to v0.10.1? Our thinking was to only ship it with v0.11.0-alpha.1 and onwards, as those are the versions with the list arithmetic rewrites that people will need.
